examples/kubernetes/1.7/cilium-crio.yaml

```diff
@@ -236,22 +241,27 @@ spec:
         - name: cilium-run
           hostPath:
             path: /var/run/cilium
+            type: "DirectoryOrCreate"
```

Comment:
@aanm are these changes expected?

Reply:
@abhide15 please ignore those changes in 1.7 manually for now. We will drop support for k8s 1.7 in 1.3

Reply:
Fixed